internal/kafka/internal/services/kafka.go

```diff
@@ -498,6 +499,28 @@ func (k *kafkaService) ListByStatus(status ...constants.KafkaStatus) ([]*dbapi.K
 	return kafkas, nil
 }
 
+func (k *kafkaService) ListKafkasToBePromoted() ([]*dbapi.KafkaRequest, *errors.ServiceError) {
+	dbConn := k.connectionFactory.New()
+
+	validPromotionStatuses := []string{
```

Comment:
In the `github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/dbapi` package we now have the `KafkaPromotionStatus` type with a set of methods.
Can we add a method that returns the "valid promotion statuses / promotable states" and reuse it here?